lib/u3d/unity_module.rb

```diff
@@ -0,0 +1,132 @@
+## --- BEGIN LICENSE BLOCK ---
+# Copyright (c) 2016-present WeWantToKnow AS
```

Comment:
I wondered if we should have used 2019 here, but we just moved code from other files in here. So 2016 sounds fine.

Reply:
My thoughts as well!